src/components/components/Image.jsx

```diff
@@ -0,0 +1,3 @@
+export function UserImage({ source }, props) {
```

Comment:
You could put this in one of the folders. Maybe services, actions, or I can move it into the Profile folder I'm currently making.